rest/api.go

```diff
@@ -226,10 +227,19 @@ func Serve(listen string, zk ZooNode) {
 	r.HandleFunc("/v2{path:[A-Za-z0-9-_/.:]+}", zk.RM).Methods("DELETE")
 	r.HandleFunc("/v2{path:[A-Za-z0-9-_/.:]+}", zk.UP).Methods("PUT", "POST", "PATCH")
 
-	http.Handle("/", r)
+	c := cors.New(cors.Options{
```

Comment:
This is for the same domain or for a '*'?

Maybe we can make it more configurable?